Makefile

```diff
@@ -101,11 +102,21 @@ install: manifests helm install-crd ## Install CRDs into a cluster
 install-crd:
 	kubectl apply -f config/crd/bases/mongodbcommunity.mongodb.com_mongodbcommunity.yaml
 
-install-chart:
+install-chart: uninstall-crd
```

Comment:
before being able to install the helm chart we need to make sure that crds are not around. They will be installed by the chart anyway